### PR TITLE
CSI Cluster Registry and Node Info CRDs Improvements

### PR DESCRIPTION
**Pull request by saad-ali**
**What this PR does / why we need it**:
https://github.com/kubernetes/kubernetes/pull/67803 merged before I could address @lavalamp's feedback. This PR addresses his feedback

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Follow up on PR https://github.com/kubernetes/kubernetes/pull/67803

**Special notes for your reviewer**:

**Release note**:

```release-note

```

/assign @lavalamp 
/assign @thockin 

CC @jsafrane @vladimirvivien @verult @gnufied @childsb

### CODE REVIEW

File: pkg/features/kube_features.go [hunk 1]
```diff
@@ -201,9 +201,9 @@ const (
 
 	// owner: @saad-ali
 	// alpha: v1.12
-	//
+	//f
```
Comment (dixudx) [inline pkg/features/kube_features.go]: What is this `f` for? Remove it?

**Comment by dims** [comment]
/release-note-none

/milestone v1.12

Looks like this addresses (or is a follow up) for 1.12

**Comment by lavalamp** [comment]
/lgtm
/approve

any chance I can still get you to switch the CRD install code to directly specify a YAML string?

**Comment by saad-ali** [comment]
> any chance I can still get you to switch the CRD install code to directly specify a YAML string?

Too late for this release (code freeze today). But will definitely do this for beta next quarter.

**Comment by hoegaarden** [comment]
@lavalamp @saad-ali 

> any chance I can still get you to switch the CRD install code to directly specify a YAML string?

Reading `[...] YAML string [...]` makes me wonder if you want
- just a variable in a go file holding the yaml content

or
- a yaml **file** that which will then be converted into go source via go-bindata or such

We started working on adding the validation bits in https://github.com/kubernetes/kubernetes/pull/68406.

@mariantalla

**Pull request merged**